test/plugins/derivedIdentity/DerivedIdentityClient.test.ts

```diff
@@ -0,0 +1,186 @@
+import { Keypair } from '@solana/web3.js';
+import test, { Test } from 'tape';
+import { amman, killStuckProcess, metaplex } from '../../helpers';
+import {
+  Metaplex,
+  derivedIdentity,
+  keypairIdentity,
+  KeypairIdentityDriver,
+  sol,
+  isEqualToAmount,
+  isLessThanAmount,
+  isGreaterThanAmount,
+} from '@/index';
+
+killStuckProcess();
+
+const init = async (
+  options: {
+    message?: string;
+    identityAirdrop?: number;
+    derivedAirdrop?: number;
+  } = {}
+) => {
+  const mx = await metaplex({ solsToAirdrop: options.identityAirdrop });
+
+  mx.use(derivedIdentity());
+
+  if (options.message != null) {
+    await mx.derivedIdentity().deriveFrom(options.message);
+  }
+
+  if (options.derivedAirdrop != null) {
+    await amman.airdrop(
+      mx.connection,
+      mx.derivedIdentity().publicKey,
+      options.derivedAirdrop
+    );
+  }
+
+  return mx;
+};
+
+const getBalances = async (mx: Metaplex) => {
+  const identityBalance = await mx.rpc().getBalance(mx.identity().publicKey);
+  const derivedBalance = await mx
+    .rpc()
+    .getBalance(mx.derivedIdentity().publicKey);
+
+  return { identityBalance, derivedBalance };
+};
+
+test('[derivedIdentity] it derives a Keypair from the current identity', async (t: Test) => {
+  // Given a Metaplex instance using the derived identity plugin.
+  const mx = await init();
+
+  // When we derive the identity using a message.
+  await mx.derivedIdentity().deriveFrom('Hello World');
+
+  // Then we get a Signer Keypair.
+  t.true(mx.derivedIdentity().publicKey);
```

Comment:
More common to use `t.ok` and t.notOk`, but those synonyms work fine.
However the failure + success messages will be very non-descriptive.

I'd recommend adding a comment string to the assertion similar to:

```ts
t.true(mx.derivedIdentity().publicKey, 'signer keypair')
```